tests/DescriptionTest.php

```diff
@@ -21,4 +21,19 @@ public function it_should_be_creatable_from_an_udb3_model_description(): void
 
         $this->assertEquals($expected, $actual);
     }
+
+    public function it_should_return_value_with_to_string(): void
```

Comment:
I'm not sure if these tests get executed with the `@test` comment or without `test` in the name.